defi/Aave/AAVE whitepaper V1.md

```diff
@@ -0,0 +1,189 @@
+# AAVE whitepaper V1
+
+受迫于周六的分享:laughing: ，故需要先把AAVE 的V1版的白皮书整理出来。先不要求一下子就完全理解AAVE中的设计取舍，按照白皮书的目录，逐项阅读，逐项理解，把自己的一点浅薄的理解写出来，也好过最后看完了，理解完了，跟人一说就这里不确定，那里不明白。:fish:
+
+
+## 介绍
+
+![image-20210916140901104](./image/image-20210916140901104.png)
+
+AAVE是一个著名的借贷协议，其源自之前的ETHlend项目，将P2P的借贷模式改为P2Protocol模式。跟Compound的借贷机制有点类似。笔者之前研习过compound的借贷机制，这里会对比下compound和AAVE在借贷机制的相同之处和不同之处。
+
+### 基础概念
+
+AAVE中的借贷池的核心是金库的概念，即每一个资金池都持有数总不同Token的余额，这些所有Token以ETH计价则得到该资金池的总流动性。每一个金库接受流动性提供者的存款，同时为借款人提供要借出的Token。借款人需要先在金库中有一定的存款，然后以这笔存款作为抵押，获得另一种或者同种Token作为借款。
+
+#### LTV: `loanToValue`:
+
+每一个金库都有一个特定的LTV，即总贷款与总价值的比率，是由该金库中每一个Token的LTV经过加权平均得到。
+
+>这里与compound比较的话，Compound也会有一个借贷池，即cToken。用户向某一个cToken池子中提供Token获得相应的cToken. 当用户需要借贷的时候，compound会计算出用户所持有的所有不同种类的cToken的价值，根据此时预言机拿到的Token的价格来计算，然后判断此时用户的流动性如何，是否允许借贷。而从AAVE的白皮书来看，它的一个reserve里包含多种Token，即一个借贷池由多种Token组成。用户提供某一种Token到这个reserve里? 不确定是否理解正确
+
+#### 清算
+
+>compound中的清算机制是用来保护cToken资金池出现系统性风险，及时将用户资不抵债的部分清算掉，并给清算者提供奖励。
+
+![image-20210916142714339](./image/image-20210916142714339.png)
+
+清算通常来自于被抵押Token的价格波动，通常是向下波动，导致用户的抵押品价值低于债务的价值，如图E点。此时超过的部分就会被清算。
+
+#### 清算阈值Lq
+
+这里的清算阈值Lq需要进一步看代码明确含义。
+
+#### 账户健康度
+
+AAVE提出了一个账户健康度的概念，其实就是一个账户的总抵押品价值与清算阈值的乘积与总债务与债务产生的费用之和的比例，即：
+$$
```

Comment:
You might need a hack for showing LaTeX formulas in GitHub markdown.md. See usage in this: https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b